documentation/api/v1/credential_api_doc.rb

```diff
@@ -167,6 +167,13 @@ module CredentialApiDoc
         end
       end
 
+      response 401 do
+        key :description, 'Authenticate to access this resource.'
```

Comment:
I know I didn't do it for the response 500 description, but we should change this to a constant that lives in `root_api_doc.rb` and reference it in each model. Can you please make the change for 401. And if you're feeling ambitious, 500, too?

Reply:
Are you talking about just description string? I couldn't find a way to make the 401s and 500s default responses without explicitly including them in every file. The string I can do, though!

Reply:
Yeah, just the description string. I actually spent way too much time trying to figure out how to make those responses default, but ultimately couldn't get it to work. This appears the be the best solution for now.